README.md

```diff
@@ -99,60 +99,60 @@ You can get it from http://repo1.maven.org/maven2/org/glassfish/javax.json/1.0.4
 
 ```
 
-When the javax.json-1.0.4.jar, commons-codec-1.1.jar and maximo-restclient-0.1.jar is ready, add them to the java project as common reference libraries.
+When the javax.json-1.0.4.jar, commons-codec-1.1.jar, and maximo-restclient-0.1.jar files are ready, add them to the Java project as common reference libraries.
 	
 # III. Usage
 -----
 
-Maximo Resources (Object Structures) represent a graph of related Maximo objects (Mbo's) that provides an atomic view/object to create/update/delete/query the releated set of Mbos. 
+Maximo Resources, or object structures, represent a graph of related Maximo business objects (Mbos) that provides an atomic view/object to create, update, delete, and query the releated set of Mbos. 
 
-We will use Work Order, Purchase Order and Companies Resource as examples to show you how to use the Maximo Rest Client.
+We will use the Work Order, Purchase Order and Companies Resources as examples to show you how to use the Maximo REST Client.
 
->**note**: The use cases can be find at TestOSLCApi.java
+>**Note**: The use cases can be found at TestOSLCApi.java.
 
-## 3.1 Query a Work Order  for Work Order Set (mxwodetail)
+## 3.1 Query a work order for work order set (mxwodetail)
 
-The following instruction shows how to query a work order from Maximo by using Maximo Rest Client Library.
+The following instruction shows how to query a work order from Maximo Asset Management by using the Maximo RET Client Library.
 
-### 3.1.1 Connect to Maximo
+### 3.1.1 Connect to Maximo Asset Management
 
-Before we connect to Maximo, it is necessary to set up authentication and environment information in Options;
+Before you connect, it is necessary to set up the authentication and environment information in Options.
```

Comment:
line 119,
the Options resource? The Options oject? For clarity, it would help to have an article and identifing noun here.